packages/gatsby/src/utils/start-server.ts

```diff
@@ -259,6 +259,14 @@ module.exports = {
 
     if (enableRefresh && authorizedRefresh) {
       refresh(req, pluginName)
+      res.status(200)
+      res.setHeader(`content-type`, `application/json`)
+    } else {
+      res.status(404)
```

Comment:
Is it worth distinguishing between disabled and unauthorised with a 403?

Reply:
For our use case not but doesn't harm us